packages/type-utils/src/TypeOrValueSpecifier.ts

```diff
@@ -146,6 +146,22 @@ function typeDeclaredInFile(
   );
 }
 
+function typeDeclaredInPackage(
+  packageName: string,
+  declarationFiles: ts.SourceFile[],
+): boolean {
+  const typesPackageName =
+    '@types/' +
+    (packageName[0] === '@'
+      ? packageName.substring(1).replace('/', '__')
+      : packageName);
+  return declarationFiles.some(
+    declaration =>
+      declaration.fileName.includes(`node_modules/${packageName}/`) ||
+      declaration.fileName.includes(`node_modules/${typesPackageName}/`),
+  );
```

Comment:
[Refactor] Just to remove the need for two iterations over the string... if you dislike this I think it's totally reasonable to ignore the suggestion though 😄 

```suggestion
  const matcher = new RegExp(
    `node_modules/(?:${packageName}|${typesPackageName})/`,
  );

  return declarationFiles.some(declaration =>
    matcher.test(declaration.fileName),
  );
```

Reply:
Thanks!